Workbooks/ZeroTrust(TIC3.0).json

```diff
@@ -0,0 +1,9698 @@

```

Comment:
Will these links work in air gapped?
Also, you need the links to be not language specific (remove the en-us)